editor/editor_node.cpp

```diff
@@ -5456,6 +5460,7 @@ EditorNode::EditorNode() {
 	p->add_separator();
 
 	p->add_item(TTR("Open Project Data Folder"), RUN_PROJECT_DATA_FOLDER);
+	p->add_item(TTR("Open Project Resource Folder"), RUN_PROJECT_FOLDER);
```

Comment:
I originally called this "Open Project Folder" but I guess this naming is more descriptive/accurate.